src/utils/save_game/save_game.cpp

```diff
@@ -0,0 +1,55 @@
+#include "save_game.h"
+#include "../utils.h"
+#include "../../creature/main_character/main_character.h"
+#include <iostream>
+#include <fstream>
+
+void SaveGame(MainCharacter& main_characher)
+{
+	std::ofstream outputFile;
+
+	outputFile.open("settings.txt");
+
+	if (outputFile.is_open())
+	{
+		outputFile << "Ability Scores for: " << std::endl;
+		outputFile << main_characher.get_name() << std::endl;
+		outputFile << "==============================" << std::endl;
+		outputFile << "HP: " << std::endl;
+		outputFile << main_characher.get_hp() << std::endl;
+		outputFile << "AC: " << std::endl;
+		outputFile << main_characher.get_ac() << std::endl;
+		outputFile << "Speed: " << std::endl;
+		outputFile << main_characher.get_speed() << std::endl;
+		outputFile << "Strength: " << std::endl;
```

Comment:
Let's cut down on some code and prepare this for future changes.
Instead of having each ability score written separately like that, put the titles ("Strength: ", "Dexterity: " etc) in their own array, then have a for loop to save these and their respective scores to the file.